README.md

```diff
@@ -52,11 +52,12 @@ The version of the extension for the Opera browser can be found at the [Opera ad
 To get a local copy of the current code, clone it using git:
 
     $ git clone git://github.com/mozilla/pdf.js.git pdfjs
-    $ cd pdfjs
+    $ cd pdf.js
```

Comment:
That seems fine to me too. I always use just `git clone git://github.com/mozilla/pdf.js.git` personally and therefore my working folder has always been `pdf.js`. It would make the command somewhat easier and I can't think of any problems. Objections, other ideas anyone?